### ISSUE TITLE
Not prompted when doing a git checkout

### ISSUE THREAD

**Issue by strboul** [rgburke/grv#84]
grv does not prompt a message box when wanting to checkout a `HEAD` state that the action is invoked by the `c` key. Setting the option `set confirm-checkout true` (which should be `true` by default anyway according to the documentation) either in the application or grvrc file did not help at all.

Could you help solve that problem?

**Comment by rgburke** [comment]
Which version of GRV are you using? Do you have any key mappings specified in `grvrc`?

I've tried with the latest static binary available [here](https://github.com/rgburke/grv/releases/download/v0.3.0/grv_v0.3.0_linux64) and it seems to work:

![image](https://user-images.githubusercontent.com/7596018/50119563-0919cf80-024b-11e9-95cf-f3f8109c7c5b.png)

**Comment by strboul** [comment]
To be clear, I mean to get a prompt when doing a git checkout in the `Status view` e.g. when returning a file to its `HEAD` state. I mistakenly typed `c` instead of `C` on a file and it led to the loss of some data!

grv version:
```
GRV - Git Repository Viewer v0.3.0-0-g5589a31 (compiled with go1.10.4 at 2018-09-29 11:55:05 BST)
```
`grvrc` file:
```
set commit-graph true
set mouse true
```

**Comment by rgburke** [comment]
Ah OK, I see what you mean. I will fix this.

**Comment by rgburke** [comment]
This should now be fixed in the latest version of GRV (v0.3.1)
